### DISCUSSION TITLE
Gate io from petra wallet transaction fail

### DISCUSSION THREAD

**Discussion by aydinsen** [aptos-labs/aptos-developer-discussions#179] [Questions]
I cannot send my thl token from gateio to petra wallet and I get an error message like this from gateio. 'invalid address. destination Address need to activate first' how can I fix this?

**Comment by aydinsen** [comment]
By the way, I have no problems sending or receiving aptos tokens.

**Comment by kshitijc1506** [comment]
Just to clarify - you have sent APT to _that_ account before (the one you tried sending THL tokens to)? I want to make sure that this isn't an "account doesn't exist on-chain" error. 

If you realize that the issue is that the account doesn't exist yet, you have two options:
1. You can send a small amount of APT to that account to create the account on-chain.
2. You can just import their gate.io account to their Petra wallet, then NO sending would be necessary.

**Reply by aydinsen** [reply]
I definitely sent aptos token to the wallet and sent aptos to gate io.

**Reply by aydinsen** [reply]
Petra has a considerable amount of aptos in her wallet.
I understand you, but my problem is that I cannot send THALA tokens. I have no problems with Aptos.

**Reply by kshitijc1506** [reply]
Spoke to someone from the Petra team and here's their take:

If they cannot send _from_ [gate.io](http://gate.io/), then it's a [gate.io](http://gate.io/) issue. (It sounds like the account doesn't exist on-chain issue we've seen in the past, but this would be a bad error on the [gate.io](http://gate.io/) - restrictive instead of helpful).

**Reply by aydinsen** [reply]
gate io uses this wallet number for some reason! Cannot identify for THL. I did not understand too.
Is there a place to register this wallet or get approval (I'm asking for my Petra account)

**Reply by kshitijc1506** [reply]
I'm geo-blocked from gate.io, I would recommend reaching out to their customer support. If they think it's a Petra issue, do let me know! Thanks :)
